trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/DummyGravitinoConnector.java

```diff
@@ -20,8 +30,85 @@
  */
 public class DummyGravitinoConnector implements Connector {
 
-  public DummyGravitinoConnector() {
+  private final CatalogConnectorManager catalogConnectorManager;
+
+  private final Set<Procedure> procedures = new HashSet<>();
+
+  public DummyGravitinoConnector(CatalogConnectorManager catalogConnectorManager) {
     super();
+    this.catalogConnectorManager = catalogConnectorManager;
+
+    try {
+      // call gravitino.system.createCatalog(metalake, catalog, provider, properties, ignoreExist)
+      MethodHandle createCatalog =
+          MethodHandles.lookup()
+              .unreflect(
+                  DummyGravitinoConnector.class.getMethod(
+                      "createCatalog",
+                      String.class,
+                      String.class,
+                      String.class,
+                      String.class,
+                      boolean.class))
+              .bindTo(this);
+
+      List<Procedure.Argument> arguments =
+          List.of(
+              new Procedure.Argument("METALAKE", VARCHAR),
+              new Procedure.Argument("CATALOG", VARCHAR),
+              new Procedure.Argument("PROVIDER", VARCHAR),
+              new Procedure.Argument("PROPERTIES", VARCHAR, false, "{}"),
```

Comment:
Can it be Map type?

Reply:
I think the map type in Trino a bit challenging to use, the usage is as follows:
```
call gravitino.system.createCatalog('test', 'jdbc-mysql7', 'jdbc-mysql', MAP(ARRAY['jdbc-url', 'jdbc-user', 'jdbc-password','jdbc-dri
ver'], ARRAY['jdbc:mysql://192.168.164.4:3306?useSSL=false', 'mysql', 'mysql', 'com.mysql.cj.jdbc.Driver']));
```
Using a JSON string might align better with our preferences.
```
call gravitino.system.createCatalog('test', 'jdbc-mysql', 'jdbc-mysql', '{"jdbc-url":"jdbc:mysql://192.168.164.4:3306?useSSL=false","jdbc-user":"mysql","jdbc-password":"mysql", "jdbc-driver":"com.mysql.cj.jdbc.Driver"}');
```

Reply:
I found that in https://iceberg.apache.org/docs/latest/spark-procedures/#examples-8, there're some call procedures that have required properties as map type. So I'm inclined to use map type. Though it is not so straight to write, but the definition of map type is much more clear than a vague varchar.